test_scripts/Defects/8_2/3931/common.lua

```diff
@@ -0,0 +1,134 @@
+            m.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", {})
+        end)
+    end
+    if pParams.vrCommands then
+        m.getHMIConnection():ExpectRequest("VR.AddCommand", {
+            cmdID = pParams.cmdID,
+            vrCommands = pParams.vrCommands
+        })
+        :Do(function(_, data)
+            m.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", {})
+        end)
+
+        table.insert(m.generatedGlobalProperties.vrHelp, { position = (#m.generatedGlobalProperties.vrHelp + 1), text = pParams.vrCommands[1] })
+        table.insert(m.generatedGlobalProperties.helpPrompt, { text = pParams.vrCommands[1], type = "TEXT" })
+        m.getHMIConnection():ExpectRequest("UI.SetGlobalProperties", {
+        vrHelp = m.generatedGlobalProperties.vrHelp
+        })
+        :Do(function(_, data)
+        m.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", {})
+        end)
+        m.getHMIConnection():ExpectRequest("TTS.SetGlobalProperties", {
+        helpPrompt = m.generatedGlobalProperties.helpPrompt
+        })
+        :Do(function(_, data)
+        m.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", {})
+        end)
+    end
+    
+    m.getMobileSession():ExpectResponse(cid, { success = true, resultCode = "SUCCESS" })
+end
+  
+function m.SetGlobalProperties(pParams)
+    local cid = m.getMobileSession():SendRPC("SetGlobalProperties", pParams)
+    if pParams.vrHelp or pParams.vrHelpTitle then
+        m.getHMIConnection():ExpectRequest("UI.SetGlobalProperties", {
+            vrHelp = pParams.vrHelp,
+            vrHelpTitle = pParams.vrHelpTitle
+        })
+        :Do(function(_, data)
+            m.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", {})
+        end)
+    end
+    if pParams.helpPrompt then
+        m.getHMIConnection():ExpectRequest("TTS.SetGlobalProperties", {
+            helpPrompt = pParams.helpPrompt
+        })
+        :Do(function(_, data)
+            m.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", {})
+        end)
+    end
+    m.getMobileSession():ExpectResponse(cid, { success = true, resultCode = "SUCCESS" })  
+end
+  
+function m.ResetGlobalProperties(pParams)
+    local cid = m.getMobileSession():SendRPC("ResetGlobalProperties", pParams)
+    if isItemInArray("VRHELPITEMS", pParams.properties) or isItemInArray("VRHELPTITLE", pParams.properties) then
+        m.getHMIConnection():ExpectRequest("UI.SetGlobalProperties", {
+            vrHelp = m.generatedGlobalProperties.vrHelp,
+            vrHelpTitle = m.defaultINIGlobalProperties.vrHelpTitle
+        })
+        :Do(function(_, data)
+            m.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", {})
+        end)
+    end
+    if isItemInArray("HELPPROMPT", pParams.properties) then
+        m.getHMIConnection():ExpectRequest("TTS.SetGlobalProperties", {
+            helpPrompt = m.defaultINIGlobalProperties.helpPrompt
```

Comment:
Help prompt should match generated as well

Reply:
Never mind, I see that would conflict with the existing INI configuration for the help prompt, suppose we'll just have to deal with this weird behavior for now